Sources/Solana/Models/Account.swift

```diff
@@ -69,36 +79,34 @@ public struct Account: Codable, Hashable {
     }
 }
 
-public extension Account {
-    struct Meta: Decodable, CustomDebugStringConvertible {
-        public let publicKey: PublicKey
-        public var isSigner: Bool
-        public var isWritable: Bool
+public struct AccountMeta: Decodable, CustomDebugStringConvertible {
```

Comment:
Should `AccountMeta` conform to `Account` protocol?

Reply:
Should not. Since its just a way to represent on the transaction.